osu.Game.Rulesets.Osu/Skinning/Legacy/LegacySliderBall.cs

```diff
@@ -79,22 +85,19 @@ protected override void UpdateAfterChildren()
 
         private void updateStateTransforms(DrawableHitObject obj, ArmedState _)
         {
-            if (obj is not DrawableSlider)
```

Comment:
As mentioned, please add this back with a comment explaining why.

Reply:
Sorry that's my bad, didn't see the reply until after I pushed.